src/MessagingInput/MessagingInput.less

```diff
@@ -18,15 +22,52 @@
 
   &.TextArea--placeholder-shown {
     .TextArea--input {
-      color: @neutral_gray;
+      color: @messaging_gray;
       min-height: @size_l;
       .text--medium();
       white-space: nowrap;
     }
   }
 }
 
+.MessagingInput--SendIcon {
+  display: none;
+}
+
+.MessagingInput--SendText {
+  display: block;
+}
+
 button.MessagingInput--SendButton {
   // Sizing for this button is pretty particular
-  padding: 0.6875rem @size_l 0.6875rem;
+  padding: 0.625rem @size_l 0.625rem;
+}
+
+@media only screen and (max-width: @breakpointM) {
+  .MessagingInput--TextField {
+    .margin--right--none();
+    border-radius: @size_m;
+    min-height: 2rem;
+  }
+
+  .MessagingInput--SendIcon {
+    display: block;
+    background-color: transparent;
+    border-radius: 50%;
+  }
+
+  .MessagingInput--SendText {
+    display: none;
+  }
+
+  button.MessagingInput--SendButton {
```

Comment:
This looks good more just have a question for my learning when you have time - how come the absolute positioning is needed if its in the Flexbox that would usually push it to the end of the flexbox container?

Reply:
Because I don't need it _after_ the TextArea (as in desktop styling), but _inside_ the TextArea (as in mobile styling).

Reply:
Ahh yes. Makes sense!

Reply:
I'm sure there are more elegant ways to do it (I don't really love using `absolute` positioning...), but it's pretty iffy when dealing with Dewey components. Not gonna go add a prop for an end-of-textarea button, after all.